src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- You did well creating an action type 👍, but according to the project requirements, the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like const CHECK_STATUS = categories/categories/CHECK_STATUS. Please fix this 👍